package_details:
  package_colour:
    packagen00: red
    packagen01: yellow
    packagen02: green
    packagen10: green
    packagen11: yellow
    packagen12: red
    packagen20: green
    packagen21: red
    packagen22: yellow
    packagen30: yellow
    packagen31: green
    packagen32: red
